docs/source/usage/plugins/countPerSupercell.rst

```diff
@@ -14,7 +14,7 @@ The plugin is available as soon as the :ref:`libSplash and HDF5 libraries <insta
 .cfg files
 ^^^^^^^^^^
 
-By specifying the perodicity of the output using the comand line argument ``--e_macroParticlesPerSuperCell.period`` (here for an electron species ``e``) with picongpu, the plugin is enabled.
+By specifying the periodicity of the output using the command line argument ``--e_macroParticlesPerSuperCell.period`` (here for an electron species ``e``) with picongpu, the plugin is enabled.
 Setting ``--e_macroParticlesPerSuperCell.period 100`` adds the number of all electron like macro particles to the file ``e_macroParticlesCount.dat`` for every 100th time step of the simulation.
```

Comment:
*electron-like* ... might be misleading with multiple electron species, though

Reply:
and the comma is unnecessary

Reply:
yes, `-like` is confusing.

Reply:
removed both the comma and the like 
fixed